tests/integration/express-server.spec.ts

```diff
@@ -476,6 +476,69 @@ describe('Express Server', () => {
       }, 400);
     });
 
+    it('should validate string-to-number dictionary body', () => {
+      const data: ValidateMapStringToNumber = {
+        key1: 0,
+        key2: 1,
+        key3: -1,
+      };
+      return verifyPostRequest(basePath + '/Validate/map', data, (err, res) => {
+        const response = res.body as number[];
+        expect(response.sort()).to.eql([-1, 0, 1]);
+      });
+    });
+
+    // TODO: explicitly nullable fields
```

Comment:
This is because I requested to allow nullable fields whatever the `property.dataType` is, while in first version @Cerberuser only allowed `any` properties to be nullable.
Here I understand that the suggestion is to later explicitly set a property to nullable with a new annotation when needed instead of allowing every property to be nullable.

Reply:
okay, but then we should probably remove this until then, because if that doesn't come, nobody will remember why it's there (: if it's just about remembering, the code could be added as a comment to this PR or something.

Reply:
Done. BTW, is there any reasoning behind keeping test fixtures for concrete servers (express, hapi, koa, custom) in repo? They are regenerated on every test anyway.

Reply:
I didn't do this, so I'm not entirely sure,  but my guess would be to make sure that the templates are still working. When running the tests, the routes are regenerated from the template and there are tests for each server to make sure that each template still works.

Reply:
Ah, sorry.. you asked why the `routes.ts` files are checked into the repository, right? That doesn't make much sense, I agree.

Reply:
ping @lukeautry